### ISSUE TITLE
Otter-Browser no Window after Starting

### ISSUE THREAD

**Issue by negout** [haikuports/haikuports#3385]
I installed Otter-Browser using HaikuDepot when I start it no window shows. I can see it listed in the applications open list in the Deskbar but that is all. Clicking on the close all button won't close it. Trying to kill it from ProcessController does not work either. The only way to kill it is from Team Monitor. Opening it from Terminal does not show any information. There is no way to uninstall it from HaikuDepot because after been installed from there it is not listed when searched again. The only way to remove it is deleting from /boot/system/packages.
I was using Release1/Beta1 hrev52295+104

**Comment by negout** [comment]
This only happens in x86_64. In Haiku x86 gcc2, Otter does show a window and mostly works.

**Comment by alaviss** [comment]
As a small tip, our package manager can also be interacted with from the command line:

    pkgman uninstall otter_browser

Would uninstall Otter Browser for you. For HaikuDepot, the "Show" menu have an option to list installed package.

**Comment by negout** [comment]
I found out what was causing Otter Browser to have no window. It was because Haiku x86_64 was not installing all the depenent files for it to work. I had updated from nightly Haiku to Haiku R1 Beta1 x86_64. When I did a complete install not over any previous nightly updates, Haiku Depot downloaded all the files dependent for Otter Browser and it starts properly showing a window. This was happening also when I had tried to install Otter Browser from previous nightlies. It was not just caused by Haiku R1 Beta1 installed over a nightly.
